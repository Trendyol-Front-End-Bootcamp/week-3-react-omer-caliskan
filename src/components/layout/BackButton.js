import React from 'react';

function BackButton(){
    return(
        <button className="backbutton">
             {"< Homepage"} 
        </button>
    );
}

export default BackButton;