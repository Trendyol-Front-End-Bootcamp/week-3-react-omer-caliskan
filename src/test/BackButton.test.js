import React from 'react'
import BackButton from "../components/layout/BackButton";
import Enzyme, { shallow } from 'enzyme';
import Adapter from "enzyme-adapter-react-16";


Enzyme.configure({ adapter: new Adapter() })

describe("BackButton Component Test", () => {
        let backbutton
  
        beforeEach(() => {
          backbutton = shallow(<BackButton />)
        })
  
        it("should render backbutton", () => {
          expect(backbutton).toMatchSnapshot();
        })
  
        it("should check backbutton class", () => {
          const buttonTextCheck = backbutton.hasClass("backbutton")
  
          expect(buttonTextCheck).toBeTruthy();
        })
      })