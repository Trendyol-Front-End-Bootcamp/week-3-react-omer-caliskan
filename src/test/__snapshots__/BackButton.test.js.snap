// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackButton Component Test should render backbutton 1`] = `ShallowWrapper {}`;
